components/shindig-server/src/main/resources/shindig.properties

```diff
@@ -225,5 +225,7 @@ shindig.jscontent.unversioned.maxage=3600
 shindig.jscontent.versioned.maxage=-1
 shindig.jscontent.invalid.maxage=0
 
+# Property to enable/disable URL whitelisting in the shindig web app.
+wso2.shindig.proxy.enabled=true
 # Whitelisting URL prefix to be used for shindig internal calls. Can provide multiple comma separated URLs.
 wso2.shindig.proxy.whitelist=
```

Comment:
I think we should add the default whitelist URL as well right?

Reply:
The default whitelist URL is already generated in the code by reading the carbon hostname from the server configs. Therefore, that would be a better approach.